include/mbedtls/ssl.h

```diff
@@ -1593,6 +1610,14 @@ struct mbedtls_ssl_context
 #endif
 };
 
+#define MBEDTLS_SSL_PEND_FATAL_ALERT( type, user_return_value )         \
+    do                                                                  \
+    {                                                                   \
+        ssl->send_alert = 1;                                            \
```

Comment:
This is not consistent with the documentation, which says the allowed values are `0`, `MBEDTLS_SSL_ALERT_LEVEL_FATAL` and `MBEDTLS_SSL_ALERT_LEVEL_WARNING`. Please either change the documentation to say `0` or `1`, or change the macro to use `MBEDTLS_SSL_ALERT_LEVEL_FATAL`.

Reply:
Thanks @mpg , I forget fix this comment